policy/modules/contrib/spamassassin.te

```diff
@@ -659,12 +660,29 @@ optional_policy(`
 	gpg_manage_home_content(spamd_update_t)
 ')
 
+optional_policy(`
+	init_status(spamd_update_t)
+')
+
+optional_policy(`
+	mta_read_config(spamd_update_t)
+')
+
+optional_policy(`
+	spamassassin_systemctl(spamd_update_t)
+')
+
+optional_policy(`
+	systemd_exec_systemctl(spamd_update_t)
+')
+
+optional_policy(`
+	userdom_search_admin_dir(spamd_update_t)
```

Comment:
userdom_search_user_home_dirs is not needed?

Reply:
this interface was only added to optional policy block and it is not related to the bz